llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -1132,7 +1132,7 @@ let OtherPredicates = [HasGFX10_BEncoding] in {
   >;
 }
 
-let SubtargetPredicate = isGFX8GFX9 in {
+let SubtargetPredicate = isGFX8GFX9NotGFX940 in {
```

Comment:
These clumsy subtarget checks should really be replaced with a subtarget feature